libsentrykube/tests/test_utils.py

```diff
@@ -1,17 +1,47 @@
 from pathlib import Path
-from unittest.mock import patch, MagicMock
-from libsentrykube.utils import get_service_registry_filepath, workspace_root
+import pytest
+from unittest.mock import patch
+from libsentrykube.utils import (
+    ensure_kubectl,
+    get_service_registry_filepath,
+    workspace_root,
+)
 
 
-@patch("importlib.import_module", return_value='/')
+@patch("importlib.import_module", return_value="/")
 @patch("importlib.resources.files", return_value=Path("a/b/c"))
 def test_get_service_registry_package_installed(
-    mock_import_module, 
-    mock_import_resources_path
+    mock_import_module, mock_import_resources_path
 ):
-    assert get_service_registry_filepath() == Path("a/b/c/sentry_service_registry/services.json")
+    assert get_service_registry_filepath() == Path(
+        "a/b/c/sentry_service_registry/services.json"
+    )
 
 
 def test_get_service_registry_package_not_installed():
     root = str(workspace_root())
-    assert get_service_registry_filepath() == Path(f"{root}/shared_config/_materialized_configs/service_registry/combined/service_registry.json")
\ No newline at end of file
+    assert get_service_registry_filepath() == Path(
+        f"{root}/shared_config/_materialized_configs/service_registry/combined/service_registry.json"
+    )
+
+
+def test_ensure_kubectl_existing_file(tmp_path):
+    with patch(
+        "libsentrykube.utils.ensure_libsentrykube_folder", return_value=tmp_path
+    ):
+        version = "1.30.4"
+        binary = "kubectl"
```

Comment:
Why don't you reuse the `KUBECTL_BINARY` and `KUBECTL_VERSION` constants defined in libsentrykube/utils.py? That way, if those constants ever change, this test would not need to change.

Reply:
Thats what I get for trying to have cursor generate my tests, good call.